lib/fiber/local/inheritable.rb

```diff
@@ -0,0 +1,41 @@
+
+require 'fiber'
+require 'securerandom'
+
+class Fiber
+	module Local
+		module Inheritable
+			def initialize(**options)
+				super(**options)
```

Comment:
You need to use `ruby2_keywords` here to properly delegate on Ruby < 3:
https://eregon.me/blog/2021/02/13/correct-delegation-in-ruby-2-27-3.html

Reply:
That's the advantage of perfect delegation which `ruby2_keywords` achieves for Ruby 2+, we don't need to know what the called method takes, it passes the arguments as if there wasn't the intermediate call.